pkg/awsvpc/sg_analysis.go

```diff
@@ -0,0 +1,228 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package awsvpc
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/ipblock"
+	"github.com/np-guard/models/pkg/netp"
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/commonvpc"
+)
+
+type SpecificAnalyzer struct {
```

Comment:
Better have `SG` and perhaps also `aws` embedded in the name. "Specific" is not very specific :-)

Reply:
by SpecificAnalyzer interface
it should be here, will change the name to AWSanalyzer, and same in ibm pkg

Reply:
I would change it to `AWSSGAnalyzer` and `IBMSGAnalyzer`

Reply:
changed